app/views/pages/faq.html.erb

```diff
@@ -3,5 +3,140 @@
+            <p class="about_p"><strong>Can I can sync my Archive-It account with the Archives Unleashed Cloud without using Twitter or Github?</strong></p>
+            <p class="about_p">Right now authentication is only offered via <%= link_to('Twitter', 'https://twitter.com/', target: '_blank') %> and <%= link_to('GitHub', 'https://github.com/', target: '_blank') %> - we hope to expand this in the future. We've had some of our colleagues come to the Cloud without an account for either Twitter or GitHub, and if you're in a similar situation we have recommended creating  a research based Twitter/Github account in order to use the Cloud. Don't worry – you don't need to code or tweet!</p>
+            <p class="about_p"><strong>Why are some collections listed as public, and others are not?</strong></p>
+            <p class="about_p">There are a variety of metadata fields that are imported in from the Archive-It system that describe collections you can work with in the Cloud. One of those fields indicates whether a collection is public or not, which is determined by users within the Archive-It system.</p>
+            <p class="about_p"><strong>Why is my collections dashboard empty?</strong></p>
+            <p class="about_p">You haven't entered your Archive-It credentials and updated your account. This will allow the Cloud to import your collections from Archive-It, and once that process is complete you will see the dashboard populated with some basic information about all of your collections.</p>
+          </div>
+        </div>
+      </div>
+      <div class="auk_faq_accord card" id="accordion">
+        <div class="auk_faq_accord-header card-header" id="headingTwo">
+          <h5 class="mb-0">
+            <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
+              Data Analysis & Visualization
+            </button>
+          </h5>
+        </div>
+        <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
+          <div class="auk_faq_accord_body card-body">
+            <p class="about_p"><strong>I've asked the system to process my collection, but the screen is still empty. How long does it take to analyze the collection?</strong></p>
+            <p class="about_p">Our Cloud works on a queue system, which means that as soon as you click “Analyze” collection, it has entered into a queue of all the jobs users are running. We operate on a first-in-first-out system, so somebody may have started processing their collection before you. Only when the analysis is done will you see derivatives.</p>
```

Comment:
"as soon as you click "Analyze" collection, it has entered into"
->
"as soon as you click "Analyze Collection," it has entered into"